api/src/main/java/org/openmrs/CohortMember.java

```diff
@@ -34,13 +34,18 @@
     public CohortMember() {
     }
 
-	public CohortMember(Integer cohortMemberId, Integer cohortId, Integer patientId, Date startDate, Date endDate) {
+	public CohortMember(Integer cohortMemberId, Cohort cohortId, Patient patientId, Date startDate) {
 		this();
 	    this.cohortMemberId = cohortMemberId;
 		this.cohortId = cohortId;
 		this.patientId = patientId;
 		this.startDate = startDate;
-		this.endDate = endDate;
+	}
+
```

Comment:
Actually you should have a constructor that takes in only a Patient object, when you add the member to a cohort  the cohort field will get set and the start date will default to current date time.